include/engine/datafacade/contiguous_internalmem_datafacade.hpp

```diff
@@ -1129,6 +1129,27 @@ class ContiguousInternalMemoryDataFacade<MLD> final
     {
     }
 };
+
+template <typename Callback>
+void ForEachSourceNodes(ContiguousInternalMemoryDataFacade<MLD> const & facade,
```

Comment:
You mean you will get a compile error because `routing_base_mld.hpp` does not include `contiguous_internalmem_datafacade.hpp` hence this will reference an unknown type?

Does including `contiguous_internalmem_datafacade.hpp` cause errors itself?